conf/machine/juno.conf

```diff
@@ -27,3 +27,7 @@ SERIAL_CONSOLE = "115200 ttyAMA0"
 
 # Juno can boot with UEFI or U-Boot
 #EXTRA_IMAGEDEPENDS = "edk2"
+UBOOT_MACHINE = "vexpress_aemv8a_juno_defconfig"
+
+MACHINE_ESSENTIAL_EXTRA_RDEPENDS = "u-boot-juno atf-juno"
+EXTRA_IMAGEDEPENDS = "u-boot-juno atf-juno"
```

Comment:
Since u-boot depends on atf, is it really needed to specify it here?

Reply:
I'm not sure if there's any black magic (or of any other color) happening around MACHINE_ESSENTIAL_EXTRA_RDEPENDS due to the nature of do_deploy() in ATF. No harm done if it's there, so it can be merged in the meantime, but I'll dig around to see if that's used somehow. If it's not, I'll update the PR.